src/Frontend/tsconfig.app.json

```diff
@@ -13,6 +13,7 @@
       "@/*": ["./src/*"],
       "@component-test-utils": ["./test/utils.ts"]
     },
+    "noUncheckedIndexedAccess": false,
```

Comment:
This is required because of [this](https://github.com/vuejs/tsconfig/releases/tag/v0.8.0) change to the latest version of @vue/tsconfig